eta/core/learning.py

```diff
@@ -472,11 +473,12 @@ class VideoClassifier(Classifier):
     that featurizes the frames of the input video.
     '''
 
-    def predict(self, video_path):
+    def predict(self, video_reader):
```

Comment:
Isn't there some code in the concrete subclasses that needs to be modified accordingly?  Or are there currently no concrete subclasses of `VideoClassifier`?

Reply:
There are no concrete subclasses.